### PR TITLE
Update Readme file

### PR DESCRIPTION
**Pull request by casperjureskog**
**PT Story:** https://www.pivotaltracker.com/story/show/142774665

**Changes proposed in this PR:**

# SloFo

This is midcourse project for CraftAcademy Class Feb-april 2017.
It's a slow food internet order page.

**Pivotal Tracker**
[Pivotal Tracker for SloFo](https://www.pivotaltracker.com/n/projects/1996373).

**GitHub**
This site base reside on [GitHub Pages](https://github.com/CraftAcademy/slow_food_online_feb_17).

### Installing

Fork and clone repo, then run:

```
$ bundle install
```
Then:

```
$rake db:migrate
```

### Deployment

We are using [Semaphoreci CI](https://semaphoreci.com)  for continous integration and [Heroku](http://heroku.com/)
The site address [https://slowfood-online-feb-17.herokuapp.com](https://slowfood-online-feb-17.herokuapp.com)

## Information about project:

* Ruby version
  Ruby 2.3.3

* Configuration
  - Ruby on rails
  - Haml

* Database creation
  - rails db:create

* Database initialization
  - rails db:migrate

* How to run the test suite
* Tests:
  - **rake** to run all test
  - **rspec** to run spec test
  - **cucumber** to run feature test

* Dependencies:
  - acts as shoppingcart gem

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,54 @@
...
+
+### Deployment
+
+We are using [Semaphoreci CI](https://semaphoreci.com)  for continous integration and [Heroku](http://heroku.com/)
+The site address [https://slowfood-online-feb-17.herokuapp.com](https://slowfood-online-feb-17.herokuapp.com)
+
+## Information aboute prodject:
 
 * Ruby version
+  Ruby 5.0.2
```
Comment (sigu) [inline README.md]: This is actually the version of rails we are using, not ruby

File: README.md [hunk 2]
```diff
@@ -1,21 +1,54 @@
...
 
 * System dependencies
+  acts_as_shopping_cart  https://github.com/crowdint/acts_as_shopping_cart
 
 * Configuration
```
Comment (sigu) [inline README.md]: Title doesnt match the content

**Review by sigu — CHANGES_REQUESTED** [review]

**Comment by sigu** [comment]
You need to cover only the important things in the Readme file
1. What is this project about?
2. What is the development and deployment process?
3. How can someone contribute to the project?
4. How does one run tests?
5. What are some of the basic things before submitting a pull request that must be done? e.g all tests must be passing

This list is not in any particular order, it just gives you the idea of what is to be included in the file. 

Have a look at the following Readme files
https://github.com/AgileVentures/WebsiteOne
https://github.com/AgileVentures/asyncvoter-api

File: README.md (hunk 1 above)
Comment (casperjureskog) [reply README.md]: Ah true

File: README.md (hunk 2 above)
Comment (casperjureskog) [reply README.md]: hopp fully it will be :-)

File: README.md [hunk 3]
```diff
@@ -1,24 +1,49 @@
...
+[Pivotal Tracker for SloFo](https://www.pivotaltracker.com/n/projects/1996373).
+
+**GitHub**
+This site base reside on [GitHub Pages](https://github.com/CraftAcademy/slow_food_online_feb_17).
+
+### Installing
+
+Fork and clone repo, then run:
+
+```
```
Comment (sigu) [inline README.md:17]: just for your information you can always add the type of hightlighting you want for the code for example in this scenario you can add the word `bash` after the triple backticks ``` bash. Not a blocker at all

**Comment by tochman** [comment]
Any action on this @casperjureskog

File: README.md (hunk 3 above)
Comment (casperjureskog) [reply README.md:17]: im not sure what you men :-/

**Review by fottander — APPROVED** [review]
Good so far.

**Pull request merged**